### ISSUE TITLE
Crash on opening new editor window?

### ISSUE THREAD

**Issue by aes-encoding** [processing/processing#5060]
I spontaneously got this when using the processing ide (ver 3.2.1) on windows 7 64bit. The window told me to report this so here I am...

![crash](https://cloud.githubusercontent.com/assets/15825490/25933541/a67800ac-35d4-11e7-96fd-c897af797f1e.png)

**Comment by benfry** [comment]
Any idea why it's not possible to write to your user directory? Is your machine in a corporate or school environment that's locked down in some fashion?

**Comment by aes-encoding** [comment]
Nope, this is my home computer. I've never seen this before even though I've been using it everyday for a few months.

**Comment by benfry** [comment]
Really odd… Looks like something is causing the `recent.txt` file to be locked or otherwise unwritable. Try renaming, moving, or deleting `C:\Users\Vladimir\AppData\Roaming\Processing\recent.txt` and see if that fixes the issue.
